dashboard/test/ui/features/signingIn.feature

```diff
@@ -56,12 +56,15 @@ Scenario: Join non-existent section from sign in page shows error
   And I type "9999999999" into "#section_code"
   And I click selector "#section_form .btn-primary"
   Then I wait until I am on "http://studio.code.org/courses"
+  Then I wait to see ".alert-danger"
   And element ".alert-danger" contains text "Could not find a section with code"
```

Comment:
This is great.  [Possible alternative](https://github.com/code-dot-org/code-dot-org/blob/87a467b23bfe8f44e31682df9a1801ca55b416ae/dashboard/test/ui/features/step_definitions/steps.rb#L574-L576):
```gherkin
And element ".alert-danger" eventually contains text "Could not find a section with code"
```